cmd/mr/scripts/test-mr.sh

```diff
@@ -199,7 +199,14 @@ timeout -k 2s 180s ../../worker/mrworker ../../apps/early_exit/early_exit.so &
 # `jobs` ensures that any completed old processes from other tests
 # are not waited upon
 jobs &> /dev/null
-wait -n
+
+sum=`expr $(pgrep -c mrworker) + $(pgrep -c mrcoordinator)`
+newSum=`expr $(pgrep -c mrworker) + $(pgrep -c mrcoordinator)`
+
+while [ $sum -eq $newSum ]
```

Comment:
Here add a comment, check if any background process has exited or not.
proc_count decreases by one when a process exits.